lib/prepend/index.js

```diff
@@ -0,0 +1,5 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function prepend(elem, list) {
+  return [elem].concat(list)
+})
```

Comment:
use `var`, `for(;;)` and speed it up, please
or at least check it would be faster

Reply:
`concat` is much faster than `for`:

`for`:
<img width="321" alt="aleksandrbabkin_aleksandrs-macbook-pro____documents_nanoutils" src="https://user-images.githubusercontent.com/25172855/38032976-8c4d6b88-32a7-11e8-824b-a8f6716d8dcc.png">

<img width="475" alt="index_js_ _prepend_ ___documents_nanoutils" src="https://user-images.githubusercontent.com/25172855/38033118-edfeba08-32a7-11e8-85c0-2f1f2c7f8966.png">

Reply:
Guys, great news!
We can improve performance using `Array` constructor and setting values with `arr[i] = list[i]`
it's even faster than `concat`